android/src/main/kotlin/com/kiwi/reversegeocode/RNReverseGeocodeManager.kt

```diff
@@ -65,6 +65,11 @@ class RNReverseGeocodeManager (reactContext: ReactApplicationContext) : ReactCon
     return formattedAddresses
   }
 
+  @ReactMethod
+  fun isPresent(callback: Callback) {
+    callback.invoke(Geocoder.isPresent())
```

Comment:
To follow the convention for invoking callbacks, could you instead do a try-catch like in the method for searchForLocations?
That means you will have something like `callback.invoke(err, res)` instead of just `callback.invoke(res)`.